tasks/install_rhel.yml

```diff
@@ -4,35 +4,41 @@
 # validate www.postgresql.org (or probably any other source).
 
   - block:
-      - name: PostgreSQL | Install all the required dependencies | yum
+      - name: PostgreSQL | Install all the required dependencies (yum) | RHEL
         yum:
-          name: ["ca-certificates","python-psycopg2", "python-pycurl", "glibc-common","epel-release","libselinux-python"]
+          name: "{{ postgresql_packages }}"
           state: present
+          update_cache: yes
 
-      - name: PostgreSQL | Add PostgreSQL repository | yum
+      - name: PostgreSQL | Add PostgreSQL repository | RHEL
         yum_repository:
            name: postgresql
            description: postgresql yum repo
            baseurl: "{{ postgresql_yum_repository_baseurl }}"
            gpgkey: "{{ postgresql_yum_repository_gpgkey }}"
+           enabled: yes
         when: postgresql_install_repository
 
-      - name: PostgreSQL | Make sure the dependencies are installed | yum
-        yum:
-         name: ["python-psycopg2", "python-pycurl", "glibc-common","libselinux-python"]
-         state: present
-         update_cache: yes
+      - name: PostgreSQL | Disable postgresql module (necessary for RHEL8+)
```

Comment:
Please add in comments why wit must be disabled

Reply:
I added a comment and a link,  NOTE there is an alternative way to fix this issue, by updating the `.repo` file, this way was the method chosen by the upstream repo